PlayCover/en.lproj/Localizable.strings

```diff
@@ -130,7 +130,7 @@
 "soundAlert.messageText" = "Incorrect Audio Settings Detected!";
 "soundAlert.informativeText" = "Your current output device does not have a sample rate of 48 or 44.1 KHz! Crashes may occur. Would you like to change your current output device's sample rate to 48 KHz?";
 "soundAlert.successText" = "Current output device sample rate set to 48 KHz";
-"soundAlert.failureText" = "Failed to set current output device's sample rate to 48 KHz. You can manually set sample rate in ‘Audio MIDI Setup’ app";
+"soundAlert.failureText" = "Failed to set current output device's sample rate to 48 KHz. You can manually set sample rate in the ‘Audio MIDI Setup’ app";
```

Comment:
Can you remove the quotes around `Audio Midi Setup`

Reply:
Yup sorry, wasn't paying attention to GitHub while I was coding